internal/config/config.go

```diff
@@ -229,13 +242,13 @@ func InitConfig(configAndStacksInfo ConfigAndStacksInfo) error {
 	ProcessedConfig.StackConfigFilesAbsolutePaths = stackConfigFilesAbsolutePaths
 	ProcessedConfig.StackConfigFilesRelativePaths = stackConfigFilesRelativePaths
 
-	fmt.Println()
-
 	if stackIsPhysicalPath == true {
-		color.Cyan(fmt.Sprintf("Stack '%s' is a directory, it matches the stack config file %s",
-			configAndStacksInfo.Stack,
-			stackConfigFilesRelativePaths[0]),
-		)
+		if g.LogVerbose {
+			color.Cyan(fmt.Sprintf("Stack '%s' is a directory, it matches the stack config file %s",
```

Comment:
it's a wording issue, fixed it.
The CLI first checks folders if it can find the files provided by the stack name using Globs.
If it does not find anything, it will search for config files using `context` and the provided `stack_name_pattern`.
In this case, the file is in the top-level folder, but the Glob does not know it since it finds it anyway.

Updated the messages to be simply:

```
The stack '%s' matches the stack config file %s
or
The stack '%s' matches the stack name pattern '%s'
```